src/pages/Bets/components/BetTable.tsx

```diff
@@ -66,12 +66,14 @@ type RowProps = {
   onClick?: () => void;
 };
 const Row = ({ betData, onClick }: RowProps) => {
+  const rowStyle = () => {
```

Comment:
we have the `classnames` package in this repo - i think we should use that to achieve this, rather than using this function